napari/layers/surface/surface.py

```diff
@@ -355,6 +360,42 @@ def shading(self, shading):
             self._shading = Shading(shading)
         self.events.shading(value=self._shading)
 
+    @property
+    def wireframe(self):
+        return self._wireframe
+
+    @wireframe.setter
+    def wireframe(self, wireframe):
+        if wireframe is None:
+            self._wireframe.reset()
+        elif isinstance(wireframe, SurfaceWireframe):
+            self._wireframe = wireframe
+        elif isinstance(wireframe, dict):
+            self._wireframe = SurfaceWireframe(**wireframe)
+        else:
+            raise ValueError(
+                f'wireframe should be a dict or SurfaceWireframe, got {type(wireframe)}'
```

Comment:
Why you do not point here that `None` is acceptable?

Reply:
oops!